core/src/main/scala/org/apache/spark/storage/DiskStore.scala

```diff
@@ -58,14 +57,6 @@ private[spark] class DiskStore(blockManager: BlockManager, diskManager: DiskBloc
     PutResult(bytes.limit(), Right(bytes.duplicate()))
   }
 
-  override def putArray(
-      blockId: BlockId,
-      values: Array[Any],
-      level: StorageLevel,
-      returnValues: Boolean): PutResult = {
-    putIterator(blockId, values.toIterator, level, returnValues)
```

Comment:
The fact that these `putArray()` methods just turned around and called `putIterator()` suggests to me that this isn't a terribly useful method to expose.